server/models/gifModel.js

```diff
@@ -14,8 +14,20 @@ module.exports = () => {
             PRIMARY KEY (gif_id),
             FOREIGN KEY (user_id) 
             REFERENCES users (id)
-            )`
-        );
+            );
```

Comment:
why creating this in multiple places, i dont understand??

Reply:
It was with this tesing that i was able to discover why the models did not work before. Buh i have been able to create a table now without that migration. but now inside models. so i don't know if i should delete the migration folder now that i can use the models?